libraries/updateclient/UpdateClient.class.php

```diff
@@ -187,12 +187,17 @@ public function print_admin_scripts() {
 
 		// Only need this JS/CSS on the plugin admin page and updates page.
 		if ($screen->base === 'plugins' || $screen->base === 'plugin-install') {
-			// This will make the jQuery below work with various languages.

```

Comment:
All of the changes in this file need to be submitted to the main Update Manager repo instead. I'll be doing this separately but wanted to go ahead and publish them here meanwhile. Not sure if we should keep these changes in this PR or wait until we get a new version of Update Manager with them included?